### ISSUE TITLE
Feature request: show top X processes

### ISSUE THREAD

**Issue by rayr007** [yujitach/MenuMeters#42]
Would it be easy to add process information to the menus?

- network show top X processes using network (strip output from nettop command)
- cpu show top X processes using CPU (strip output from top command)
- memory show top X processes using memory (top command output again)
- disk show top X processes actively reading/writing to disk?

**Comment by yujitach** [comment]
Yes it won't be difficult but I don't have the time, unfortunately. I hope someone would contribute the code so that I can merge. 

Two requests to those who would implement the features: 

1.  I like the current minimal interface, so please do *not* make the detailed info the default. Please also add the toggle switch to enable/disable it in the pref pane. 

2.  I just merged a nice patch reducing the energy use significantly. Please do not poll the info too excessively to degrade the energy efficiency.

**Comment by yujitach** [comment]
It's implemented in v1.9.7, thanks to @HofiOne .

**Comment by rayr007** [comment]
From what I can see, only second point of this was implemented. Network, memory and disk meters do not have top X processes feature, only CPU.

**Comment by yujitach** [comment]
Ah, I see. Reopened.

**Comment by yujitach** [comment]
The same issue is posted recently, so I'm updating this.

**Comment by rozhl** [comment]
My workflow suddenly uses up all the available memory. Closing and reopening the affected app helps in my case. So knowing which app is currently eating the RAM helps. I second this feature.

I see a comment above regarding excessive polling - wouldn’t be enough to just pull the data the moment you click on the menubar icon? I mean I don’t care about the info unless I need it...
